subiquity/common/filesystem/boot.py

```diff
@@ -125,7 +125,7 @@ class SetAttrPlan(MakeBootDevicePlan):
 
     device: object
     attr: str
-    val: str
+    val: Union[str, bool]
```

Comment:
Really I think Any is sort of more appropriate here. (I guess it has to be something that can be serialized as yaml and /probably/ `Union[str,bool,int]` would cover anything this is ever likely to be but...)

Reply:
Yes, that's more sensible.